v2/pkg/cli/executor.go

```diff
@@ -190,15 +190,48 @@ func NewMirrorCmd(log clog.PluggableLoggerInterface) *cobra.Command {
 	cmd.Flags().BoolVar(&opts.Global.StrictArchiving, "strict-archive", false, "// If set, generates archives that are strictly less than `archiveSize`, failing for files that exceed that limit.")
 	cmd.Flags().StringVar(&opts.Global.SinceString, "since", "", "Include all new content since specified date (format yyyy-MM-dd). When not provided, new content since previous mirroring is mirrored")
 	// nolint: errcheck
-	cmd.Flags().MarkHidden("v2")
 	cmd.Flags().AddFlagSet(&flagSharedOpts)
 	cmd.Flags().AddFlagSet(&flagRetryOpts)
 	cmd.Flags().AddFlagSet(&flagDepTLS)
 	cmd.Flags().AddFlagSet(&flagSrcOpts)
 	cmd.Flags().AddFlagSet(&flagDestOpts)
+	HideFlags(cmd)
 	return cmd
 }
 
+func HideFlags(cmd *cobra.Command) {
+	cmd.Flags().MarkHidden("v2")
+	cmd.Flags().MarkHidden("authfile")
```

Comment:
Hey Alex, according to what customers requested, we need to keep this one

Reply:
Yes, we can remove this MarkHidden for authfile when we make sure this is working as expected in CLID-29.